nested_set.go

```diff
@@ -137,7 +137,7 @@ func moveToRightOfPosition(tx *gorm.DB, targetNode nodeItem, position, depthChan
 		targetWidth := targetRight - targetLeft + 1
 
 		targetIds := []int64{}
-		err = tx.Where(formatSQL(":rgt >= ? AND :rgt <= ?", targetNode), targetLeft, targetRight).Pluck("id", &targetIds).Error
+		err = tx.Where(formatSQL(":lft >= ? AND :rgt <= ?", targetNode), targetLeft, targetRight).Pluck("id", &targetIds).Error
```

Comment:
please tell me why if here we do need **LFT**
test works fine here after change.